src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html

```diff
@@ -38,6 +38,8 @@
                  i18n>Value</label>
           <div class="cd-col-form-input">
             <input id="value"
+                   (focus)="valueFocus.next($any($event).target.value)"
+                   (click)="valueClick.next($any($event).target.value)"
```

Comment:
Can you try the following to fix the problem with overflow:
```suggestion
                   (click)="valueClick.next($any($event).target.value)"
                   container="body"
```

Reply:
Thanks that worked :)